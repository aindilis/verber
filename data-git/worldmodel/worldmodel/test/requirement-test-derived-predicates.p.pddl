(define
 (problem REQUIREMENT-TEST-DERIVED-PREDICATES)
 (:domain REQUIREMENT-TEST-DERIVED-PREDICATES)
 (:objects obj2 obj1 - type1)
 (:init
  (pred1 obj1))
 (:goal
  (and
   (pred1 obj2))))