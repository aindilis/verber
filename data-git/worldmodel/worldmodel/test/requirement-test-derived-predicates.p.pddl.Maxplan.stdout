MaxPlanII beta.

bb: parsing domain file